package.json

```diff
@@ -29,10 +29,10 @@
 		"reflect-metadata": "^0.1.13",
 		"retire": "^2.2.5",
 		"semver": "^7.3.4",
-		"ts-node": "^8",
+		"ts-node": "^10",
```

Comment:
3pp for ts-node https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qEE0000002417YAA/view

Reply:
Approved